app/controllers/items_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ItemsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @item = Item.new
```

Comment:
since you have initialized a new item as an instance variable, maybe use that in the form?